Dockerfile

```diff
@@ -38,6 +38,7 @@ RUN pip install -U poetry
 
 ARG INSTALL_DEV_DEPENDENCIES=false
 COPY pyproject.toml poetry.lock $APP_HOME/
+RUN poetry lock
```

Comment:
should be removed

Reply:
Ah sorry I forgot to remove it. It's removed now.